ci/pipelines.release.yml

```diff
@@ -72,6 +72,7 @@ pipelines:
               jf rt mvn clean install -U -B
             - jf rt bag && jf rt bce
             - jf rt bp
+            - jf rt u --flat=true target/teamcity-artifactory-plugin-$NEXT_VERSION.zip ecosys-oss-release-local/org/jfrog/teamcity/teamcity-artifactory-plugin/$NEXT_VERSION/
```

Comment:
Let's move the upload command 2 lines up, below the mvn command. This is important to include it in the build info.